### PR TITLE
add validation tags to image notebooks

### PR DESCRIPTION
**Pull request by sharma-riti**
# Description
add validation tags to image notebooks

# Checklist

- [x] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ ] I have coordinated with the docs team (mldocs@microsoft.com) if this PR deletes files or changes any file names or file extensions.
- [ ] Pull request includes test coverage for the included changes.
- [ ] This notebook or file is added to the [CODEOWNERS](https://github.com/Azure/azureml-examples/blob/main/.github/CODEOWNERS) file, pointing to the author or the author's team.

### CODE REVIEW

**Review by PhaniShekhar — APPROVED** [review]

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb [hunk 1]
```diff
@@ -1381,7 +1478,10 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "name": "deploy"
+    "name": "deploy",
```
Comment (bhimar) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb:1481]: why does this one have "name"?

File: sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb [hunk 2]
```diff
@@ -87,7 +87,10 @@
      "transient": {
       "deleting": false
      }
-    }
+    },
+    "tags": [
```
Comment (bhimar) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb:91]: for my understanding, how are these metadata tags used?

File: sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb [hunk 3]
```diff
@@ -87,7 +87,10 @@
...
+     "validation-workspace"
+    ]
```
Comment (bhimar) [inline sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb:93]: for my understanding how are these tags (referring to all of them not these in particular) used?

**Review by bhimar — APPROVED** [review]

File: sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb (hunk 1 above)
Comment (sharma-riti) [reply sdk/python/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items.ipynb:1481]: object detection ones are used int our docs.. that is why they have names... :)

File: sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb (hunk 3 above)
Comment (sharma-riti) [reply sdk/python/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb:93]: discussed offline

**Pull request merged**